### PR TITLE
More DNS tests

### PR DESCRIPTION
**Pull request by shastrinator**
Add expiry and restore cases for the included record types.

Signed-off-by: Kiran Shastri <shastrinator@gmail.com>

### CODE REVIEW

File: agent-ovs/ovs/DnsManager.cpp [hunk 1]
```diff
@@ -152,6 +156,8 @@ namespace opflexagent {
         }
         if(isCName()) {
             if(currTime >= cachedCName.expiryTime) {
+                LOG(DEBUG) << domainName << "->" << cachedCName.cName << " expired " <<
```
Comment (tomflynn) [inline agent-ovs/ovs/DnsManager.cpp:159]: Do we need to keep these debug logs?

File: agent-ovs/ovs/DnsManager.cpp [hunk 2]
```diff
@@ -729,6 +735,7 @@ namespace opflexagent {
             bool regularMatch = DomainNameMatch(entry.domainName, demandItr.first);
             if(regularMatch || entry.matchesAliases(demandItr.first, matchingAlias)
                || entry.matchesSrv(demandItr.first, matchingSrv)) {
+                LOG(DEBUG) << entry.domainName << " matched " << demandItr.first;
```
Comment (tomflynn) [inline agent-ovs/ovs/DnsManager.cpp:738]: same debug log question...this seems like it could be printed often

**Review by tomflynn — APPROVED** [review]

File: agent-ovs/ovs/DnsManager.cpp (hunk 1 above)
Comment (shastrinator) [reply agent-ovs/ovs/DnsManager.cpp:159]: This one I think will be useful in debugging.

File: agent-ovs/ovs/DnsManager.cpp (hunk 2 above)
Comment (shastrinator) [reply agent-ovs/ovs/DnsManager.cpp:738]: I am on the border on this one,  will remove.

**Pull request merged**